BREAKING: update documentation by setting default video: false for v13.0.0 (#5300)
cypress-io/cypress-documentation: docs/guides/end-to-end-testing/migration/protractor-to-cypress.mdx

the failure.

Cypress assists with debugging in headless mode, by automatically taking a

> screenshot of the app UI and command log at the exact point of test failure. To see everything that happened prior to test failure, enabled the `video` configuration option to record (as an MP4 file) the full test spec run.

Author: ```suggestion
screenshot of the app UI and command log at the exact point of test failure. To see everything that happened prior to test failure, enable the `video` configuration option to record (as an MP4 file) the full test spec run.
```

the failure.

Cypress assists with debugging in headless mode, by automatically taking a

> screenshot of the app UI and command log at the exact point of test failure. If
> enabled, to help see everything that happened prior to test failure, Cypress
> provides a video recording (as an MP4 file) of a full test spec run by default.

Reviewer: ```suggestion
screenshot of the app UI and command log at the exact point of test failure. To see everything that happened prior to test failure, enabled the `video` configuration option to record (as an MP4 file) the full test spec run.
```
this isn't by default anymore
